package.json

```diff
@@ -150,8 +150,8 @@
 	"scripts": {
 		"prebuild": "npm run check-engines",
 		"clean:packages": "rimraf ./packages/*/build ./packages/*/build-module ./packages/*/build-style",
-		"prebuild:packages": "npm run clean:packages && lerna run build && cross-env INCLUDE_PACKAGES=babel-plugin-import-jsx-pragma,postcss-themes,jest-console SKIP_JSX_PRAGMA_TRANSFORM=1 node ./bin/packages/build.js",
-		"build:packages": "cross-env EXCLUDE_PACKAGES=babel-plugin-import-jsx-pragma,jest-console,postcss-themes node ./bin/packages/build.js",
+		"prebuild:packages": "npm run clean:packages && lerna run build",
+		"build:packages": "node ./bin/packages/build.js",
```

Comment:
Looks like a nice simplification, why don't we need the separate build configs anymore?

Reply:
Got it, no transpilation needed for these packages, should we add the minimum node version for jest-console too?

Reply:
`jest-console` still uses transpilation but we can add `node >= 8` for consistency 👍 

This makes me wonder if we still need to use Babel for node based packages seeing how well ESNext features are supported in node >= 10:
https://node.green/

Reply:
It turned out we have many packages where we didn't have this node version set. I fixed all of them in 78208c6.

Reply:
> It turned out we have many packages where we didn't have this node version set. I fixed all of them in [78208c6](https://github.com/WordPress/gutenberg/commit/78208c67e4e6e8fd695983396ca6507a0f957940).

If we want this for all packages, can we enforce it via `npm-package-json-lint`?

Reply:
I picked only those packages which are executed from CLI. We probably can identify those packages as they are listed in the root `package.json` inside `devDependencies` section. If that makes sense, I would be fine enforcing it with a rule.